recipe/bld-onednn.bat

```diff
@@ -2,11 +2,17 @@
 
 md "%SRC_DIR%"\build
 pushd "%SRC_DIR%"\build
-set TBBROOT=%PREFIX%
 set CMAKE_PREFIX_PATH=%PREFIX%
+if %dnnl_cpu_runtime%==tbb (
+    set TBBROOT=%PREFIX%
+    set DNNL_CPU_RUNTIME=TBB
+    )
+if %dnnl_cpu_runtime%==omp set DNNL_CPU_RUNTIME=OMP
+if %dnnl_cpu_runtime%==threadpool set DNNL_CPU_RUNTIME=THREADPOOL
```

Comment:
Use `[…]` quotes on Windows for better stability.
```suggestion
if [%dnnl_cpu_runtime%] == [tbb] (
    set TBBROOT=%PREFIX%
    set DNNL_CPU_RUNTIME=TBB
    )
if [%dnnl_cpu_runtime%] == [omp] set DNNL_CPU_RUNTIME=OMP
if [%dnnl_cpu_runtime%] == [threadpool] set DNNL_CPU_RUNTIME=THREADPOOL
```

Reply:
Added brackets, thanks!